### DISCUSSION TITLE
I cannot sign up vercel in Azerbaijan

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#3036] [Help] [answered]
### Summary

The website needs verification number when i enter it says your number is blocked when i use another number this time says too many requests i did a couple of times and wait for 2 days but still not works

### Steps to Reproduce

..

**Comment by swarnava** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Reply by ghost** [reply]
Thanks but now i use another platform to share next.js app
